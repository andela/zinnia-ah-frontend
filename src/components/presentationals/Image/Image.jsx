import React from 'react';
import PropTypes from 'prop-types';

const Image = ({ src, ...props }) => {
  return <img src={src} {...props} />;
};

Image.propTypes = {
  src: PropTypes.string.isRequired,
  props: PropTypes.object,
};

export default Image;
