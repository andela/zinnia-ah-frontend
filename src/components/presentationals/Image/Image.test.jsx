import React from 'react';
import { shallow } from 'enzyme';

import Image from './Image';

const src = 'https:myImage.jpg';

const snapshot = shallow(<Image src={src} />);

describe('<Image />', () => {
  it('should render correctly', () => {
    expect(snapshot).toMatchSnapshot();
  });

  it('should receive the correct props', () => {
    expect(snapshot.props().src).toBe(src);
  });
});
