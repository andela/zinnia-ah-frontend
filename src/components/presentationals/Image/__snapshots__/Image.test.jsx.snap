// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Image /> should render correctly 1`] = `
<img
  src="https:myImage.jpg"
/>
`;
